guides/lumen-supply-metrics.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Lumen Supply Metrics
+---
+
+[SDF’s Dashboard API endpoint](https://dashboard.stellar.org/api/v2/lumens) will always have the up-to-the-second totals for the essential numbers around lumens. This guide explains important supply metrics like Original Supply, Total Supply, and Circulating Supply entailed in that data. 
```

Comment:
Will the totals be `up-to-the-second`?  Or are they, I don't know, `up-to-the-ledger totals`? May be a nitpick, but feels like we're being exact here.

Reply:
updated to "live"